tests/LinAlg/vectorTestsRajaPar.hpp

```diff
@@ -76,7 +76,7 @@ class VectorTestsRajaPar : public VectorTests
   virtual ~VectorTestsRajaPar(){}
 
 private:
-  virtual const real_type* getLocalDataConst(const hiop::hiopVector* x);
+  virtual const real_type* getLocalDataConst(hiop::hiopVector* x);
```

Comment:
(some) vector implementations can change, and yes, it because of the host/device memory transfer needed. still returning a const, which I think is ok for the purpose of testing...